omod/src/main/resources/apps/dashboardWidgets_app.json

```diff
@@ -33,12 +33,13 @@
       "icon": "icon-user-md",
       "label": "coreapps.clinicianfacing.healthTrendSummary",
       "concepts": "5087AAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAA,5088AAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAA,5089AAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAA",
+      "encounterType": "123AAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAA",
```

Comment:
is this line necessary?

Reply:
Without this line the "Health Trend Summary" shows “None in the last 30 days” ie. no entries get displayed. The reference OpenMRS wiki document used to set these parameters was the 'Obs Across Encounters' section of:
https://wiki.openmrs.org/display/docs/Patient+Summary+Widget+Documentation

Reply:
@HerbertYiga @dkayiwa  Also, in the above link there is a _note_ under the 'Obs Across Encounters' section which states that :
As of Coreapps v1.29.0, the column containing the date (Encounter) was changed to 'Date' 

Though it's not very clear from the document it could be that the attribute "encounterType" actually displays the date and not the encounter type (the 'date' column was earlier named as 'Encounter' and so, now the attribute name looks confusing). To display encounter type another optional attribute 'showEncounterTypeName' needs to be used.

So, the attribute "encounterType" needs to be defined in the json file to display date and without that the widget fails to show any entry. The missing attribute "encounterType" was the main reason behind the error described in this ticket.